crates/symbolicator/src/services/shared_cache.rs

```diff
@@ -135,6 +141,7 @@ impl GcsState {
     }
 
     async fn store(&self, key: SharedCacheKey, mut src: File) -> Result<SharedCacheStoreResult> {
+        let total_bytes = src.seek(SeekFrom::End(0)).await?;
```

Comment:
you have a `rewind()` right below which already existed before. Given that, what do we expect the current seek position is when calling this function? Is it at the beginning? (which means the previously `rewind()` was useless) Or is it undefined (which means we need to add another explicit `rewind()` here.

Reply:
I was treating the position before calling this as undefined, which i think is the more defensive approach.

I don't think we need a second rewind: here we explicitly go to the end to find out the size.  then explicitly go to the beginning so we can use `io::copy()`.

Reply:
Oh, I maybe misunderstood the function signature. The function will `then yield the new position in the object`, not sure why I was thinking about it returning the number of bytes that were seeked over. So I guess this was just my misunderstanding.